readme.md

```diff
@@ -362,8 +401,8 @@ let add = (x,y) => (
 Support for ES6 is great! - https://kangax.github.io/compat-table/es6/
 
 If you need to support a legacy browser, check out the following tools:
-- [Traceur](https://github.com/google/traceur-compiler/wiki/Getting-Started)
 - [Babel](https://babeljs.io/)
+- [Traceur](https://github.com/google/traceur-compiler/wiki/Getting-Started)
```

Comment:
I think Babel is significantly more popular and the likelihood of a student reading up on both isn't great